ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13.yaml

```diff
@@ -85,6 +85,10 @@ tests:
   optional: true
   steps:
     workflow: hypershift-aws-conformance
+- as: hypershift-none-ibmcloud-zvsi
```

Comment:
why is this in 4.13 branch instead of just main?

Reply:
wanted to go for 4.13 tech preview so placed it in the 4.13 release config